pkg/kademlia/dial/dialer.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (C) 2019 Storj Labs, Inc.
 // See LICENSE for copying information.
 
-package kademlia
+package dial
```

Comment:
Try to see whether you can find a better name for this.

Reply:
Any suggestions? Perhaps package `dialer`, but change the name of the variable that the dialer is assigned to, ie 
```
type Foo struct {
    dial *dialer.Dialer
}
```
Perhaps permutations of the word "dial" isn't the best way to go